src/js/components/Tree/Tree.js

```diff
@@ -0,0 +1,161 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { ThemeContext } from 'styled-components';
+import { FormDown } from 'grommet-icons/icons/FormDown';
+import { FormNext } from 'grommet-icons/icons/FormNext';
+import { FormPrevious } from 'grommet-icons/icons/FormPrevious';
+
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { List } from '../List';
+import { Text } from '../Text';
+
+const Tree = ({ children, mode, data }) => {
```

Comment:
Can the caller start from a node already selected?

Reply:
Not at the moment. Nice enhancement.